web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -260,9 +260,9 @@ export function SelectResource({ onSelect }: SelectResourceProps) {
                     </Box>
                   </Flex>
 
-                  {r.unguidedLink && (
+                  {r.unguidedLink && r.hasAccess ? (
```

Comment:
Why would someone not have access to a guide though? Is it just that we know ahead of time that they won't be able to add e.g. a Windows desktop so we want to lock the guide behind "Lacking Permissions"?

Reply:
Yeah – both guided & unguided resource cards are disabled without permission to add new resources, they both have the 'lacking permissions' tag added and clicking on them disabled.

Reply:
Come to think of it though, would it be better to _not_ disable the unguided resource cards? They just link to the relevant documentation in a new tab.